### PR TITLE
Fix Liberty support

### PR DESCRIPTION
**Pull request by alvarolopez**

### CODE REVIEW

**Comment by alvarolopez** [comment]
Unfortunately, builds are failing due to https://github.com/travis-ci/apt-package-whitelist/pull/2497 :cry:

**Comment by enolfc** [comment]
+1 
Let's see if the travis builds gets fixed

**Comment by alvarolopez** [comment]
I've changed travis to use sudo until  travis-ci/apt-package-whitelist#2497 is fixed.

File: doc/source/installation.rst [hunk 1]
```diff
@@ -16,7 +16,11 @@
 VOMS module Installation
 ========================
 
+<<<<<<< HEAD
```
Comment (enolfc) [inline doc/source/installation.rst]: this should not be here

Comment (alvarolopez) [inline doc/source/installation.rst]: Damn!

**Comment by alvarolopez** [comment]
@enolfc the conflict remainder should be fixed now.

**Pull request merged**